apps/src/templates/sectionProgress/sectionProgressRedux.js

```diff
@@ -298,12 +298,7 @@ export const loadScript = (scriptId) => {
       return fetch(url, { credentials: 'include' })
         .then(response => response.json())
         .then((data) => {
-          const dataByStudent = data.students;
-          let studentLevelProgress = {};
-          Object.keys(dataByStudent).forEach((studentId) => {
-            studentLevelProgress[studentId] = _.mapValues(dataByStudent[studentId], getLevelResult);
-          });
-          dispatch(addStudentLevelProgress(scriptId, studentLevelProgress));
+          dispatch(addStudentLevelProgress(scriptId, data.students));
```

Comment:
I don't have a great understanding of what the before/after data here looks like, but the goal of using `getLevelResult` was to make sure `sectionProgress` and `progress` are as similar as possible. If they need to be different, that's fine as long as by not filtering, we're not storing a ton of extra data in redux that we don't need.

Reply:
Before, there was `5336: 31`, where 31 was the result.  After, we have `5336 :{status: "perfect", result: 31, paired: true}`. The bulk of my change is accessing `paired` from student data based on a new action in `progressRedux`. The problem I'm running into is in trying to use the status here, rather than doing the acrobatics of going through `statusForLevel`. Basically, what I _think_ was happening is we filtered out status, then reconstructed it again, when we already had access to it before.

Reply:
It's possssible that we handle status different on the front end and back end, but if we definitely don't then this seems fine. @balderdash might know more about this.